app/services/users.js

```diff
@@ -26,3 +28,94 @@ exports.existUserMail = async mail => {
     return new Error(`signUpUser Error => ${error.message}`);
   }
 };
+
+exports.getUserByEmailAndPassword = async (mail, newPassword) => {
+  try {
+    let hasCorrectParameters = false;
+    const userData = await userModel.findOne({ where: { mail } });
+
+    if (objIsNotEmpty(userData) && userData.password) {
+      hasCorrectParameters = await checkPassword(newPassword, userData.password);
+    }
+
+    return hasCorrectParameters ? userData : {};
+  } catch (error) {
+    logger.error(`getUserByEmailAndPassword Error => ${error}`);
+    return new Error(`getUserByEmailAndPassword Error => ${error.message}`);
+  }
+};
+
+exports.getUserByAttribute = async condition => {
+  try {
+    const user = await userModel.findOne({ where: { ...condition } });
+
+    return user;
+  } catch (error) {
+    logger.error(`getUserByAttribute Error => ${error}`);
+    return new Error(`getUserByAttribute Error => ${error.message}`);
+  }
+};
+
+exports.getUserById = id => {
+  try {
+    return userModel.findByPk(id);
+  } catch (error) {
+    logger.error(`getUserById Error => ${error}`);
+    return new Error(`getUserById Error => ${error.message}`);
+  }
+};
+
+exports.getUsersList = async (options = {}) => {
+  try {
+    const returnOptions = {};
+    const conditions = {
+      ...options,
+      attributes: { exclude: ['password'] }
+    };
+
+    const users = await userModel.findAll(conditions);
+    const totalUsers = await userModel.count();
+
+    if (options.limit && options.offset) {
+      returnOptions.limit = options.limit;
```

Comment:
I think that pagination options you could to make in serializer. Also I think taht pagination could to have 

`
    count: result.rows.length,
  page: parseInt(page),
  total_count: parseInt(result.count),
  total_pages: Math.ceil(result.count / limit)
`